### DISCUSSION TITLE
Always show'NotSignedIn'！

### DISCUSSION THREAD

**Discussion by zixiangcode** [community/community#14453] [Copilot Conversations] [answered]
I've made it on to the waiting list.And I should already be able to use Copilot.But when I click sign in,it always displays **NotSignedIn**.
In addition to that, It also prompts '**A device activation code will be copied to the clipboard.Pleasepaste this on the GitHub website to authorize GitHub Copilot.**'But in fact, I didn't find anything on my sticky board.How can I solve this problem
![QQ截图20220407112558](https://user-images.githubusercontent.com/87527187/162114212-56e239cd-275f-4794-931b-672209423f8e.png)
?

**Comment by zixiangcode** [comment, marked as answer]
I had solved this problem, but then another one came along.When I successfully authorized on the authorization page, my VS didn't change at all.How to solve this problem?
![QQ截图20220407114020](https://user-images.githubusercontent.com/87527187/162115860-c9fc6f74-8029-44df-a45d-623fbe51ffae.png)

**Reply by jason-cech** [reply]
I'm having the same issue with no device code being added to my clipboard. What was your solution to that issue? @zixiangcode

**Reply by zixiangcode** [reply]
I don't know the exact right solution,but I can use copilot normally now through my method.I first shut down the VPN and used a normal Internet connection.Then restart VS and try SignIn again, the browser will automatically open the authorization page, and the authorization code will also be automatically pasted on the paste board.After the license was granted, I restarted VS and all the issues were resolved.

**Comment by jxujoe** [comment]
一样的，授权后VS2022还是没有登录上， 但是VS code 上能够使用

**Reply by zixiangcode** [reply]
已经解决了，关闭VPN后就行了，我显示授权成功后，重启了vs就好了

**Reply by jxujoe** [reply]
![image](https://user-images.githubusercontent.com/42632579/162123208-8cc92370-e107-406f-9e0e-db45e6134790.png)
![image](https://user-images.githubusercontent.com/42632579/162123229-a48fd072-b8d6-44b0-9a39-3bdbdd46e8cf.png)
我关闭后，重启也是一样的问题。你用的什么浏览器?谷歌？

**Reply by zixiangcode** [reply]
是的，用的chrome，VPN关闭之后，再点击sign in  它就会打开默认浏览器弹窗了，这时候粘贴板也有授权码了，授权成功后我vs并没有立即出现变化，然后我尝试重启vs再来一次，就成功了

**Reply by jxujoe** [reply]
我的尝试过很多次，还是都不行，在等等吧  很多人都是这个问题，谢了

**Comment by virginity-is-cool** [comment]
楼上的答案有个卵用，重启不行

**Reply by leonevil1412** [reply]
关vpn也没有用

**Reply by leonevil1412** [reply]
still NotSignedIn

**Comment by 1160708674** [comment]
我遇到了同样的问题，一直会提示“设备激活码将被复制到剪贴板。请在GitHub网站上粘贴此内容，以授权GitHub Copilot

**Reply by zixiangcode** [reply]
你试试关闭VPN后再重启vs尝试登录。我不知道其他人什么情况，反正我是这样解决的

**Reply by peterli09** [reply]
一点用都没有，我是了好几次都不行，每次都成功授权GItHub Copilot,下一步无法进行

**Comment by virginity-is-cool** [comment]
请问你对甘雨暗削怎么看？

**Reply by qzdata** [reply]
![image](https://user-images.githubusercontent.com/67967381/166852671-a5463e44-6a22-40a9-8825-050c7ecf1e93.png)
![image](https://user-images.githubusercontent.com/67967381/166852694-38708363-05db-46ae-bdcb-87481e16700f.png)
![image](https://user-images.githubusercontent.com/67967381/166852741-d1bc93a5-96cc-40c4-9ded-bf5f4c28dbf5.png)
一开始也不行，可以在EDGE中打开，填了验证码也显示成功，一直不行
1，关闭VPN
2，添加GITHUB帐户
3，重启WINDOWS
我这样操作可以了，你们可以试一下

**Comment by lhtuling** [comment]
确实是 网络问题,
我用手机卡流量 打开手机热点电脑连接后直接就可以了!
